packages/cli/src/upload/upload.js

```diff
@@ -145,13 +151,13 @@ async function postStatusToGitHub(options) {
     githubToken,
     githubAppToken,
     githubApiHost = DEFAULT_GITHUB_API_HOST,
+    githubAppUrl = GITHUB_APP_STATUS_CHECK_URL,
```

Comment:
I think you'll need to thread this option through in `runGithubStatusCheck` as well

Reply:
Right